requirements/constraints.txt

```diff
@@ -0,0 +1,22 @@
+# Version constraints for pip installation.
+#
+# This file doesn't install any packages. It specifies version constraints
+# that will be applied if a package is needed.
+#
+# When pinning something here, please provide an explanation of why.  Ideally,
+# link to other information that will help people in the future to remove the
+# pin when possible.  Writing an issue against the offending project and
+# linking to it here is good.
+
+
+# Score fetching compatible with open-release/ironwood.master
```

Comment:
Add in the comment that this is the last version compatible with django1.11

Reply:
Added like so.